elastic/filebeat/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG ES_VERSION
+
+# # get official filebeat image
+FROM docker.elastic.co/beats/filebeat:${ES_VERSION}
+
+# Logstash host and port configuration (required)
+ARG ES_LOGSTASH_HOST
+ARG ES_LOGSTASH_PORT
+
+# Set logstash host and port for configuration file
+ENV ES_LOGSTASH_HOST=${ES_LOGSTASH_HOST}
```

Comment:
You're passing these into the container through the docker-compose as well, so im not sure if it's needed here. You can probably get rid of the ARG and ENV lines